report_builder/models.py

```diff
@@ -21,9 +21,26 @@
 def get_allowed_models():
     models = ContentType.objects.all()
     if getattr(settings, 'REPORT_BUILDER_INCLUDE', False):
-        models = models.filter(model__in=settings.REPORT_BUILDER_INCLUDE)
+        allModelNames = ()
+        additionalModels = []
+        for element in settings.REPORT_BUILDER_INCLUDE:
+            splitElement = element.split('.')
+            if len(splitElement) == 2:
+                additionalModels.append(models.filter(app_label=splitElement[0], model=splitElement[1]))
+            else:
+                allModelNames = allModelNames + (element,)
```

Comment:
This is a very slow way to add items to a list. It is much more efficient to use `append` with a list than summing tuples together.